src/chat/chat_panel.rs

```diff
@@ -395,6 +395,7 @@ live_design! {
 
                     UserChatLine = <UserChatLine> {}
                     ModelChatLine = <ModelChatLine> {}
+                    EndOfChat = <View> {height: 0.1}
```

Comment:
We still need to render something real with non-zero height at the end to make this trick work.